src/xdist/dsession.py

```diff
@@ -97,17 +98,21 @@ def pytest_collection(self):
         return True
 
     @pytest.hookimpl(trylast=True)
-    def pytest_xdist_make_scheduler(self, config, log):
+    def pytest_xdist_make_scheduler(self, config, log) -> Scheduling | None:
         dist = config.getvalue("dist")
-        schedulers = {
```

Comment:
Not using the dict because I don't want to include the `__init__` in the protocol (if that's even possible...)